README.md

```diff
@@ -63,7 +63,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- N/A
+- View the app [here](https://braingenius2.github.io/To-Do-List/)
 
```

Comment:
Great job @Braingenius2 🤝

- [ ] Kindly ensure your MIT license link is specifically for this project `To-Do-List` but not `Awesome-books-with-ES6`. This will make your readme file more professional.